src/noble/group.ts

```diff
@@ -5,7 +5,7 @@
 
 import { bytesToNumberBE, bytesToNumberLE } from '@noble/curves/abstract/utils'
 
-import { type Deserializer,type  Group,type  GroupID, Groups } from '../groupTypes.js'
+import { type Deserializer, type Group, type GroupID, Groups } from '../groupTypes.js'
```

Comment:
any idea why this did not get caught at lint time by the CI test suite?

Reply:
this is a relatively new feature, likely there is no rule that catches this.